src/sagemaker/feature_store/feature_group.py

```diff
@@ -190,6 +193,7 @@ def _ingest_single_batch(
                 client to perform boto calls.
             start_index (int): starting position to ingest in this batch.
             end_index (int): ending position to ingest in this batch.
+            profile_name (str): the profile credential should be used for ``PutRecord``.
```

Comment:
Lets add default values to new args in docstrings whereever possible for explicity.

Reply:
Sure.